en/identity-server/next/docs/references/extend/user-mgt/write-a-custom-event-handler.md

```diff
@@ -0,0 +1,136 @@
+# Write a Custom Event Handler
```

Comment:
titles should be sentence case

Reply:
Fixed with 117f6a81c92f846c04e4193c37344c0df46f5e55